packages/react/src/components/Alert/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Alert from './';
+import axe from '../../axe';
+
+const defaults = { show: false, heading: { text: 'hi' } };
+
+test('should return null when passed a falsey "show" prop', () => {
+  render(
+    <Alert data-testid="alert" {...defaults}>
+      Test Alert
+    </Alert>
+  );
+  expect(screen.queryByTestId('alert')).toBeNull();
```

Comment:
This was an older test style, so we can clean this up slightly to check that it does not render when show is false:

```suggestion
  expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
```